lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,10 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p #{project.name}",
+        "cp #{project.name}.nuspec #{project.name}/",
+        "(cd #{project.name}; /cygdrive/c/ProgramData/chocolatey/choco pack)",
```

Comment:
Ah, interesting. I've added in `@shell`, but it doesn't have the functionality yet that you're implying. I do think we should look into this, but I want to get things working, and then get that in there. I think we're just about there now, I just need to fix the `chocolateyInstall.ps1` and `chocolateyUninstall.ps1` scripts